wasmcloud_host/config/prod.exs

```diff
@@ -13,6 +13,7 @@ config :wasmcloud_host, WasmcloudHostWeb.Endpoint,
   http: [:inet6, port: System.get_env("PORT") || 4000],
   # This is critical for ensuring web-sockets properly authorize.
   url: [host: "localhost", port: System.get_env("PORT")],
+  check_origin: false,
```

Comment:
It makes sense why this would fix the issue, but I don't know enough at first glance if this would cause security issues. What kinds of vulnerabilities do we introduce by not checking the origin of a request to the dashboard? (cc @jordan-rash @autodidaddict )

Reply:
the goal behind the way it was designed was to never allow remote access to this dashboard. So long as the default host on which it runs is one that is non-routable I'm okay with not checking origin

Reply:
What we can do is allow localhost & 127.0.0.1 unless it's overridden by the environment variable